specification/definitions/Shipment.json

```diff
@@ -152,6 +152,82 @@
             }
           }
         },
+        "customs": {
+          "type": "object",
+          "properties": {
+            "package_type": {
+              "type": "string",
+              "enum": [
+                "merchandise",
+                "sample_merchandise",
+                "returned_merchandise",
+                "documents",
+                "gifts"
+              ],
+              "example": "merchandise"
+            },
+            "invoice": {
```

Comment:
Misschien is `invoice` niet beschrijvend genoeg.

`invoice_number` of `invoice_id`?